README.developers.md

```diff
@@ -661,6 +667,14 @@ The following table provides details on available Koios settings in VTR flow:
 
 For more information refer to the [Koios benchmark home page](vtr_flow/benchmarks/verilog/koios/README.md).
 
+To make running all the koios benchmark easier, especially with thos circuits scattered between different tasks, we added a task list that triggers all the 40 circuits of Koios as follows (this will run all the circuits with cimplex DSP functionality enabled. If you want to disable the complex DSP, edit the file to point to `koios_*_no_hb` tasks):
+
+```shell
+$ ../scripts/run_vtr_task.py -l koios_task_list.txt 
+
+#Several hours later... they complete
+#
+
 The following steps show a sequence of commands to run the `koios` tasks on the Koios benchmarks:
```

Comment:
Change to:

If you want to run a subset of the koios benchmarks or run them without hard DSP blocks, you can run lower-level 'koios' tasks as follows: